src/mysky/index.ts

```diff
@@ -206,7 +206,19 @@ export class MySky {
 
     // Redirect if we're not on the preferred portal. See
     // `redirectIfNotOnPreferredPortal` for full load flow.
-    await MySky.instance.redirectIfNotOnPreferredPortal();
+    try {
+      // TODO: Uncomment the below line once autologin is released. Otherwise
```

Comment:
`nit` probably want to comment the entire `try-catch` block instead of its contents